docker/db-migration/migrations/initial-access-management.yaml

```diff
@@ -0,0 +1,13 @@
+---
+databaseChangeLog:
+  - changeSet:
+      id: 1
```

Comment:
I'd recommend to use descriptive ids for changesets.

Reply:
done, maybe.